### PR TITLE
update 7-slim

### PR DESCRIPTION
**Pull request by shihchang**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -180,15 +180,6 @@ pipeline {
                 }
             }
         }
-
-        stage('Publish Image') {
```
Comment (desagar) [inline Jenkinsfile:184]: suggest removing the when { not { buildingTag() } } on other stages

File: Jenkinsfile [hunk 2]
```diff
@@ -180,15 +180,6 @@ pipeline {
                 }
             }
         }
-
```
Comment (pfmackin) [inline Jenkinsfile:183]: How is this related to the tag value change?

Comment (shihchang) [reply Jenkinsfile:183]: The stage(‘PublishImage’) in our pipelines runs when a new git tag is created to tag the image with latest commitHash to the given git-tag. Reformatting the imageTag broke this as the imageTag of latest commitHash no longer exist. We can either fix it to tag the latest image or remove stage(‘PublishImage’). Kevin and MikeG prefer removing it.

File: Jenkinsfile (hunk 1 above)
Comment (shihchang) [reply Jenkinsfile:184]: if we want to remove all the whenNotBindingTag, we will have to either disallow adding a new git-tag or in jenkins pipelines describe "do nothing if bindingTag"

Comment (desagar) [reply Jenkinsfile:184]: @shihchang answered this in another PR - we still want the ability to say "do nothing if a tag is pushed", therefore those are not removed.

**Review by desagar — APPROVED** [review]
